web3/contract.py

```diff
@@ -1,2447 +0,0 @@
-"""Interaction with smart contracts over Web3 connector.
```

Comment:
Hmmm this looks like it wasn't properly rebased... let me see if I can change that